tests/reports/commands_tests.py

```diff
@@ -342,6 +342,11 @@ def create_no_alert_email_chart(request):
             "validator_type": ReportScheduleValidatorType.NOT_NULL,
             "validator_config_json": "{}",
         },
+        "alert7": {
+            "sql": "SELECT first from test_table where first=0",
```

Comment:
If this is supposed to return zero rows, would a more obvious false where clause be more descriptive? Something like `where 1 = 0`? The other examples above are easy to reason about as they have a static query that doesn't reference a table.

Reply:
good point, better readability. Updated